scripts/make_inventory_file

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+import fcntl
+import json
+import socket
+import struct
+import sys
+import yaml
+
```

Comment:
Some documentation with a usage example would be great. 
Bonus points for a -h option that outputs that documentation. :)

Reply:
After talking to Ryan on IRC I retract my -h suggestion. :) Probably doesn't make a whole lot of sense in that context since the output is piped to the script. Maybe just some documentation in the script source code?

Reply:
Fixed in the lastest